fleet/enroll/provision_test.go

```
@@ -0,0 +1,99 @@
+	b, err := p.Provision(ctx, enroll.TargetSpec{Kind: "b2", Bucket: "hody-backups", AdminKeyID: "adm", AdminKey: "sec"}, "ag_9")
+	require.NoError(t, err)
+	require.Equal(t, "agents/ag_9/", b.Prefix)
+	require.Len(t, fake.created, 2)
+	require.Equal(t, "warphold-ag_9-writer", fake.created[0].Name)
+	require.NotContains(t, fake.created[0].Capabilities, "deleteFiles")
+	require.Equal(t, "warphold-ag_9-reader", fake.created[1].Name)
+	require.NotContains(t, fake.created[1].Capabilities, "writeFiles")
+	ci, _, err := repo.DecodeToken(b.ConnectToken)
+	require.NoError(t, err)
+	require.Equal(t, "b2", ci.Type)
+	require.Equal(t, b.WriterKeyID, "kid-warphold-ag_9-writer")
```

golangci-lint (2.13.2)

[error] expected-actual: need to reverse actual and expected values

(testifylint)